tests/data/modules/PETASCE_02.for

```diff
@@ -194,13 +201,21 @@ C      CALL GET('SPAM', 'KCBMAX', KCBMAX)
       !Maximum crop coefficient (Kcmax) (FAO-56 Eq. 72)
       WND = MAX(1.0,MIN(WIND2m,6.0))
       CHT = MAX(0.001,CANHT)
-      SELECT CASE(MEEVP)
-        CASE('A') !Alfalfa reference
-            KCMAX = MAX(1.0,KCB+0.05)
-        CASE('G') !Grass reference
-            KCMAX = MAX((1.2+(0.04*(WND-2.0)-0.004*(RHMIN-45.0))
+
+      IF (MEEVP .EQ. 'A') THEN
```

Comment:
@pratikbhd Can you please create a test program that has a SELECT statement so that we know to come back and deal with it?

Reply:
Test program for this construct has been in `tests/data/cases` directory

Reply:
Thanks.  Resolved.